### ISSUE TITLE
ApiKeys: Add API key box UI layout is broken

### ISSUE THREAD

**Issue by torkelo** [grafana/grafana#27322]
Looks ok in Add Permission box (same design) for folders and dashboards but API Key box looks like this 

![Screenshot from 2020-09-02 09-08-29](https://user-images.githubusercontent.com/10999/91942907-fa639c80-ecfb-11ea-9690-7a2e48f00350.png)

**Comment by piotrzarycki** [comment]
can i take this?

**Comment by dprokop** [comment]
@piotrzarycki absolutely yes! Thanks!

**Comment by nikasvan** [comment]
Hey @piotrzarycki If you have not started working on this can I dig in? I found the bug but there are 2 ways which it can be done and I just needs some clarifications to which component to use. @dprokop @torkelo  It seems that there are 2 "X"(close) icon components. One is `<IconButton>` and another one is `<Icon>` (wrapped in `<button>` div). The reason why "X" button is not breaking in dashboard box is that it uses `<Icon>` component but API keys box uses `<IconButton>` (which needs some small additional styling). I think there is some inconsistency with "X" components. I've looked into it and have a fix for both just need a direction to which component to use here.
